subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -433,6 +694,23 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_cmd_ap,
 	SHELL_SUBCMD_SET_END
 );
 
+SHELL_STATIC_SUBCMD_SET_CREATE(wifi_twt_ops,
+	SHELL_CMD(setup, NULL, " Start a TWT flow:\n"
+		"<negotiation_type, 0: Individual, 1: Broadcast, 2: Wake TBTT>\n"
+		"<setup_cmd: 0: Request, 1: Suggest, 2: Demand>\n"
+		"<dialog_token> <flow_id> <responder> <trigger> <implicit> "
+		"<announce> <twt_wake_interval_ms> <twt_interval_ms>\n",
+		cmd_wifi_twt_setup),
```

Comment:
Can valid value ranges be indicated for different parameters (e.g. dialog_token, flow_id etc) ?